addon/components/-private/simple-checkbox.js

```diff
@@ -39,6 +38,11 @@ export default class SimpleCheckbox extends Component {
   @type(optional(Action))
   onChange = null;
 
+  @argument
+  @type('string')
+  @attribute('aria-label')
+  ariaLabel;
```

Comment:
What is this ariaLabel used for ?

Reply:
it's for the a11y test, checkboxes are now enabled based on `selectMode` so they're enabled for all tests. a11y test checks to make sure all form controls are properly labeled.